engine/engine/src/test/def-3575/def-3575-bitmap.font

```diff
@@ -1,5 +1,5 @@
 font: "/builtins/fonts/vera_mo_bd.ttf"
-material: "/builtins/fonts/system_font.material"
+material: "/builtins/fonts/font.material"
```

Comment:
Below, you corrected the material assignment in `def-3575-df.font` to use the distance field font material `font-df.material`. If you wanted to do the same here, I think this should be using the bitmap font material `font-fnt.material`? 🤔

I suspect it doesn't really matter for this test case, though.

Reply:
Done!